docs/astrocut/file_formats.rst

```diff
@@ -88,27 +106,27 @@ NAXIS4    Length of second array dimension (NAXIS2 from FFIs)
 BinTableHDU (Extension 2)
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The BinTableHDU extension contains a table that holds all of the Image extension
-header keywords from the individual FFIs. There is one column for each keyword
-plus one additional column called "FFI_FILE" that contains FFI filename for each
-row. Each column name keyword also has an entry in the extension header, with
-the value being the keyword comment from the FFI header. This last allows the
-FFI Image extension headers to be reacreated completely if desired.
+The BinTableHDU extension, in both the SPOC and TICA cubes, contains a table that 
+holds all of the Image extension header keywords from the individual FFIs. There 
+is one column for each keyword plus one additional column called "FFI_FILE" that 
+contains FFI filename for each row. Each column name keyword also has an entry in
+the extension header, with the value being the keyword comment from the FFI header.
```

Comment:
I didn't change this sentence too much from what was there originally (partially because I didn't fully follow it either), but I *think* this is saying that the column names are given as keywords in the header of the BinTableHDU. I don't really know how to interpret the "value being the keyword comment form the FFI header" part though.

Reply:
Can you format this to: 
_Each column name keyword also has an entry in the **Image** extension header, with the value being the keyword **value** from the FFI header_

And yeah, it sounds like this is just saying that each row under a column is populated by the keyword value for that keyword (column) of the FFI that the row corresponds to.